src/documentation/transxchange_read.md

```diff
@@ -71,11 +71,11 @@ NTFS field | TransXChange element | Mapping rule/Comment
 --- | --- | ---
 line_id | *Services/Service/ServiceCode* | This field is prefixed.
 line_code | *Services/Service/Lines/Line/LineName* |
-line_name | *Services/Service/Description* | If no *Description* is found, the value is computed as "backward_line_name - forward_line_name" 
+line_name | *Services/Service/Description* | If no *Description* is found, then this field is computed using the name of the first associated Route in the forward direction. If several forward routes exist, the one with the smallest `route_id` is used.
```

Comment:
> The one with the smallest `route_id` is used

Is it about the length of String? What if there is 2 routes with a `route_id` of the same length?

It feels like it doesn't matter which one we're choosing if several forward routes are found, the only constraint might be that it is consistently the same one that is chosen, am I right?

Reply:
It is the "smallest" in alphabetical order mostly, I will add a precision in the next commit. And yes, the idea is that the first trip is consistently chosen in the same way.